packages/pn-commons/src/utility/index.ts

```diff
@@ -37,6 +37,15 @@ import {
 } from './notification.utility';
 import { compileOneTrustPath } from './onetrust.utility';
 import { calculatePages } from './pagination.utility';
+import {
+  PAYMENT_CACHE_KEY,
+  checkIfPaymentsIsAlreadyInCache,
```

Comment:
If we apply the changes to the paymentCaching.utility.ts, some of these exportss will be useless